doc/source/user_guide/groupby.rst

```diff
@@ -891,18 +997,27 @@ and that the transformed data contains no NAs.
    grouped_trans.count()  # counts after transformation
    grouped_trans.size()  # Verify non-NA count equals group size
 
-.. note::
+.. _groupby_efficient_transforms:
 
-   Some functions will automatically transform the input when applied to a
-   GroupBy object, but returning an object of the same shape as the original.
-   Passing ``as_index=False`` will not affect these transformation methods.
+As mentioned in the note above, each of the examples in this section can be computed
+more efficiently using built-in methods.
 
-   For example: ``fillna, ffill, bfill, shift.``.
+.. ipython:: python
 
-   .. ipython:: python
+    # ts.groupby(lambda x: x.year).transform(
```

Comment:
On purpose?

Reply:
Yes - the idea here is that we are showing how to write more efficient transforms. The example demonstrating `transform` with a UDF from this section is commented out, and the efficient alternative is below.

I've added a comment to make this more clear.